libraries/AC_AutoTune/AC_AutoTune.cpp

```diff
@@ -19,10 +19,9 @@
  # define AUTOTUNE_LEVEL_RATE_RP_CD         500     // rate which qualifies as level for roll and pitch
 #endif
 #define AUTOTUNE_LEVEL_RATE_Y_CD            750     // rate which qualifies as level for yaw
-#define AUTOTUNE_REQUIRED_LEVEL_TIME_MS     500     // time we require the aircraft to be level
-#define AUTOTUNE_LEVEL_TIMEOUT_MS          2000     // time out for level
-#define AUTOTUNE_LEVEL_WARNING_INTERVAL_MS 5000     // level failure warning messages sent at this interval to users
-#define AUTOTUNE_ANGLE_MAX_RLLPIT          30.0f    // maximum allowable angle in degrees during testing
+#define AUTOTUNE_REQUIRED_LEVEL_TIME_MS     250     // time we require the aircraft to be level
+#define AUTOTUNE_LEVEL_TIMEOUT_MS           2000    // time out for level
+#define AUTOTUNE_ANGLE_MAX_RLLPIT           30.0f   // maximum allowable angle in degrees during testing
```

Comment:
```suggestion
#define AUTOTUNE_LEVEL_TIMEOUT_MS          2000     // time out for level
#define AUTOTUNE_ANGLE_MAX_RLLPIT          30.0f    // maximum allowable angle in degrees during testing
```

Reply:
Sorry, I don't understand what you are suggesting here. On my screen everything all likes lined up.

Reply:
It should line up on the right ride. It lines up on the left side now.

Reply:
Ahh, I see what you are saying. My convention has been to line it up on the left size on a tab indent. I will ask what everybody thinks I should be doing on the dev call tomorrow.

Reply:
Feedback is this is ok.